go/vt/vtgate/planbuilder/filter.go

```diff
@@ -0,0 +1,18 @@
+// Copyright 2016, Google Inc. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package planbuilder
+
+import "github.com/youtube/vitess/go/vt/sqlparser"
+
+// appendFilters breaks up the BoolExpr into AND-separated conditions
+// and appends them to filters, which can be shuffled and recombined
+// as needed.
+func appendFilters(filters []sqlparser.BoolExpr, node sqlparser.BoolExpr) []sqlparser.BoolExpr {
```

Comment:
It's a Go thing (short names). I think the comment is clear about what the function does.
Additionally, the append prefix is another Go thing; this mimics Go's append function, which people intuitively understand.